packages/storage-plus/src/prefix.rs

```diff
@@ -153,6 +156,96 @@ where
     }
 }
 
+#[derive(Clone)]
+pub struct Prefix2<K, T>
```

Comment:
I would look fore some more descriptive name for that....

Reply:
This is addressed in #439, where this functionality is integrated into `Prefix` directly.

I think it would be a good idea to merge that here, so that you can review everything at once before merging.

Reply:
Will look at this when I would finally finish my tests I try to do since yesterday ;) Anyway - if it is addressed there and it should be merged before this one, please rebase.

Reply:
Done.